### ISSUE TITLE
Need to be able to provide config files at startup.

### ISSUE THREAD

**Issue by daniel-england** [roribio/alpine-sqs#11]
Need to have a way to use my config files on start up instead of editing them every time and restarting a service.

**Comment by progracademy-owner** [comment]
Hi, 

Thank you for your interest :)

Could you please provide some more detail on what you're trying to accomplish? 

Keep in mind that this project is only a convenience Docker implementation of the [ElasticMQ](https://github.com/adamw/elasticmq) project.

Thanks,
Ronald

**Comment by roribio** [comment]
Hi, 

Thank you for your interest :)

Could you please provide some more detail on what you're trying to accomplish? 

Keep in mind that this project is only a convenience Docker implementation of the [ElasticMQ](https://github.com/adamw/elasticmq) project.

Thanks,
Ronald

**Comment by daniel-england** [comment]
Hi Ronald,

The only way I could find to add queues is to add them manually.  It would be nice if I could pass in the elasticmq.conf and sqs-insight.conf to use when launching the container.

**Comment by note** [comment]
@daniel-england I stumbled upon similar problem. According to `README.md`:

```
Alternatively, if you'd like to be able to edit configuration files locally, without having to enter the container, you can mount the configuration directory somewhere on your host directly.

docker run --name alpine-sqs -p 9324:9324 -p 9325:9325 -v /opt/alpine-sqs:/opt/config -d roribio16/alpine-sqs:latest
```

The problem is that it does not really work. The reason is that if I mount my directory to `/opt/config` on container it still gets overwritten by `sqs-init.sh` which does `cp /opt/*.conf /opt/config/`. One may think that it might work to mount to `/opt` instead but it does not. I don't know the semantics of mounting to already existing directories - in my case trying to do so wiped out all the existing content which of course caused everything failing.

@roribio I prepared a PR that fixes that: https://github.com/roribio/alpine-sqs/pull/14
